app/controllers/authenticate_controller.rb

```diff
@@ -128,23 +128,24 @@ def k8s_inject_client_cert
   private
 
   def handle_login_error(err)
-    logger.debug("Login Error: #{err.inspect}")
+    logger.info("Login Error: #{err.inspect}")
     err.backtrace.each do |line|
       logger.debug(line)
```

Comment:
Why do we print info level only for the first issue in the trace?

Reply:
we print just the one line that customers will look for and can help them. This line prints the stack trace which is more for developers and thus is printed at DEBUG level.